activate.py

```diff
@@ -137,7 +137,7 @@ def main(eth1_endpoint, graph_endpoint, private_key, contract_address, contract_
                           help="private key associated with the account whitelisted with pstake stketh to make the transaction",
                           required=True)
     parser.add_argument("-contract", "--contract-address", help="contract address to make the transaction to",
-                        default="0x2aDd159D38d9Dd1d980Bc017666073F91823d56d")
+                        default="0xC98a8aE7c5D009410543977a6f9CB4921741a03e")
```

Comment:
why change this default?

Reply:
contract was changed in testnet. The address was updated here itseld=f